web/html/src/manager/systems/ssm/ssm-counter.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { useWebSocket } from "../../shared/websocket/useWebSocket";
+
+type Props = {
+  count?: number;
+};
+
+export function SsmCounter(props: Props) {
+  const [count, setCount] = React.useState(props.count);
+  const [errors, setErrors] = React.useState([]);
+  useWebSocket(errors, setErrors, "ssm-count", (value: number) => {
+    setCount(value);
+  });
+  return (
+    <div id="header_selcount">
+      <span id="spacewalk-set-system_list-counter" className="badge">
+        {count}
+      </span>
+      {count === 1 ? t("system selected") : t("systems selected")}
```

Comment:
This might not be the correct way to translate this, perhaps @parlt91 or someone else who is familiar with our translation logic can chime in? The problem here is that while in English the logic is one ("system") or many ("systems"), in some other languages there are sometimes more cases (e.g. separate cases for one, two, three and then many). I'm not sure what the correct way to address this is though with our setup, hoping someone else can chime in.

Reply:
So what way should we go for this one?

Reply:
The final decision is to postpone proper pluralization to a future PR.